src/book.js

```diff
@@ -161,6 +161,14 @@ export class Book {
             this.rearrangedpages = this.book.pagelist;
         } else if (this.format == 'a9_3_3_4' || this.format == 'a10_6_10s' || this.format == 'A7_2_16s' || this.format == '1_3rd' || this.format == 'a_3_6s' || this.format == 'a_4_8s') {
             this.book = new WackyImposition(this.orderedpages, this.duplex, this.format, this.pack_pages)
+        } else if (this.format == 'partsheetsigstandard' || this.format =='partsheetsigcustom') {
```

Comment:
you introduce this format `partsheetsigstandard` and `partsheetsigcustom` but there's still no way for that to be selected...